io/deeplink/deeplink.go

```diff
@@ -0,0 +1,12 @@
+package deeplink
+
+import (
+	"net/url"
+)
+
+// Event is generated when the app is opened from a deeplink, or when the app is already running and a deeplink is opened.
+type Event struct {
+	URL *url.URL
+}
```

Comment:
This seems to belong in `io/transfer`. Not sure whether it should be `transfer.DataEvent`.

Reply:
The issue with `DataEvent` is that it uses ReadCloser. Of course, it's possible to use some `strings.NewReader(url)`, but I don't think that it's right. Maybe create a new `transfer.LinkEvent` or `transfer.URLEvent`?

Reply:
`URLEvents` seems fine for now.